besu/src/test/java/org/hyperledger/besu/cli/options/SynchronizerOptionsTest.java

```diff
@@ -51,7 +51,7 @@
         .downloaderChangeTargetThresholdByHeight(
             SynchronizerConfiguration.DEFAULT_DOWNLOADER_CHANGE_TARGET_THRESHOLD_BY_HEIGHT + 2)
         .downloaderChangeTargetThresholdByTd(
-            SynchronizerConfiguration.DEFAULT_DOWNLOADER_CHANGE_TARGET_THRESHOLD_BY_TD.plus(2L))
```

Comment:
This should remain as `plus` for Kotlin operator overloading compatibility.

Reply:
this is coming from UInt256. I can add plus as an alias to add to the UInt256 class, if that does the job?

Reply:
added an alias in UInt256.